pallets/xyk/src/lib.rs

```diff
@@ -169,11 +169,11 @@ pub mod pallet {
 		/// Liquidity was removed from the pool. [who, asset a, asset b, shares]
 		LiquidityRemoved(T::AccountId, AssetId, AssetId, Balance),
 
-		/// Pool was created. [who, asset a, asset b, initial shares amount]
-		PoolCreated(T::AccountId, AssetId, AssetId, Balance),
+		/// Pool was created. [who, asset a, asset b, initial shares amount, pool id]
+		PoolCreated(T::AccountId, AssetId, AssetId, Balance, AssetId),
 
-		/// Pool was destroyed. [who, asset a, asset b]
-		PoolDestroyed(T::AccountId, AssetId, AssetId),
+		/// Pool was destroyed. [who, asset a, asset b, pool id]
+		PoolDestroyed(T::AccountId, AssetId, AssetId, AssetId),
```

Comment:
this is share token id not exactly pool id -> we should add pool address as well

Reply:
added